cmake/projects/YAML_CPP/hunter.cmake

```diff
@@ -0,0 +1,21 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+  PACKAGE_NAME
+  YAML_CPP
```

Comment:
Ok, but in this case one should use package `yaml` for hunter package addition and find `YAML_CPP` by `find_package`. I think this is missleading. Anyway the project calls itself as `yaml-cpp` refering to the "official" `c++` implementation of yaml standard. So I think it would be better to rename the project to `yaml-cpp` and rewrite the package project name also to this.